src/render.js

```diff
@@ -28,6 +28,10 @@ async function getFiles(dir) {
   return Array.prototype.concat(...files);
 }
 
+/* wow,
+  this would need to be refactored into a different form
+  for ignoring generator actions files
+*/
```

Comment:
@anithri Seems like you don't need to refactor completely. I think adding another filter before the internal prompt/index filter. Maybe adding a glob to regex library like [`globrex`](https://www.npmjs.com/package/globrex).

Then you wouldn't need ignore configuration for each type of thing, just ignore based on path.


Something like: 
```javascript
// user config
const config = {
	userIgnores: [
		'wip/**',
		'utils.js'
	]
}

// ...
const globrex = require('globrex');
const userIgnores = config.userIgnores.map(g => globrex(g))


// ...
.then(filter(f => !userIgnores.find(ig => ig.regex.test(f))))
```

Reply:
Might make more sense to just forget the glob syntax and support plain regex